src/main/java/edu/itu/cavabunga/controller/wrapper/ParameterResponse.java

```diff
@@ -0,0 +1,62 @@
+package edu.itu.cavabunga.controller.wrapper;
+
+import edu.itu.cavabunga.core.entity.Parameter;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ParameterResponse {
+    private Integer status;
+    private String message;
+    private List<Parameter> data = new ArrayList<Parameter>();
+
+    public Integer getStatus() {
```

Comment:
do we really need the status fields in these response class? Why is HTTP status codes not enough?

Reply:
Status is changing with error codes. You may have the same 404 Http Code(which states for not found) for differ cases. Status code gives you idea that which error code you have in application to determine problem about request or operation.

Reply:
IMHO, you can use 404 for all missing component, parameter etc. Maybe I am missing something, we should redesign these response classes together.